cishouseholds/weights/pre_calibration.py

```diff
@@ -22,12 +22,21 @@ def pre_calibration_high_level(df_survey: DataFrame, df_dweights: DataFrame, df_
         on="ons_household_id",
         how="left",
     )
-    # TEMPORARY
-    for column_name in ["ever_never_swab", "ever_never_antibodies", "longcovid", "14_days", "28_days", "42_days"]:
-        df = df.withColumn(column_name, F.lit(1))
-    df = df.withColumn("index_multiple_deprivation", F.col("index_multiple_deprivation").cast("float"))
-    # TODO - define logic for ever_never_swab, longcovid, 14_days, etc.
-    # TODO - extract index_multiple_deprivation, region_code columns
+
+    # TODO: find the right column name for each of the inputs
+    df = dataset_generation(
+        df=df,
+        # these cutoff dates need to be provided
+        cutoff_date_swab="2022-02-09",
```

Comment:
These cutoff dates will change each run, so will need to be passed via a config rather than hard coded. Please could you take these as parameters from the config into the pre-cal stage?